src/plugins/GenerateJob/GenerateJob.js

```diff
@@ -177,19 +177,16 @@ define([
                 const filename = GenerateJob.toSnakeCase(name);
                 const code = this.core.getAttribute(node, 'code');
 
-                return [filename, name, code];
+                return [filename, code];
             });
 
         operations.forEach(tuple => {
-            const [filename, name, code] = tuple;
+            const [filename, code] = tuple;
 
             // Add file to `operations/`
             files.addFile(`operations/${filename}.py`, code);
```

Comment:
Should we still be adding unused files to our jobs? I don't know of any reason to keep this